Signal.xcodeproj/xcshareddata/xcschemes/Signal.xcscheme

```diff
@@ -37,10 +37,9 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Test"
```

Comment:
I previously changed test-runs to user Debug so that we could step through debug our compiled pods. I've reverted that change and solved this the "proper" way by disabling optimization per-pod in Test configuration in the Podfile.